lib/db/dao/activity_template_dao.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:async';
+import 'package:maui/app_database.dart';
+import 'package:sqflite/sqflite.dart';
+import '../entity/activity_template.dart';
+
+class ActivityTemplateDao {
+  Future<ActivityTemplate> getTopic(String id, {Database db}) async {
+    db = db ?? await new AppDatabase().getDb();
+    List<Map> maps = await db.query(ActivityTemplate.table,
+        columns: [ActivityTemplate.activityIdCol, ActivityTemplate.imageCol],
+        where: '${ActivityTemplate.activityIdCol} = ?',
+        whereArgs: [id]);
+    if (maps.length > 0) {
+      return ActivityTemplate.fromMap(maps.first);
+    }
+    return null;
+  }
+
+  Future<List<ActivityTemplate>> getAllTemplates({Database db}) async {
```

Comment:
Why would we need to get all the templates in our database? This should be getActivityTemplatesByActivityId(int activityId)